Tests/API/Api.cs

```diff
@@ -256,8 +254,8 @@ public void Download_AndSave_DataCorrectly()
             Assert.IsTrue(downloadedMinuteData);
             Assert.IsTrue(downloadedDailyData);
             
-            Assert.IsTrue(File.Exists(Config.Get("data-folder") + "forex/oanda/daily/eurusd.zip"));
-            Assert.IsTrue(File.Exists(Config.Get("data-folder") + "forex/oanda/minute/eurusd/20131011_quote.zip"));
+            Assert.IsTrue(File.Exists(_dataFolder + "forex/oanda/daily/eurusd.zip"));
```

Comment:
Perhaps its worth deleting the file if it exists already? Otherwise its might read a false positive